src/Plugin/Task/ReleaseCreateTask.php

```diff
@@ -42,7 +42,10 @@ public function run()
             'tag_name' => $arguments['tag'],
             'target_commitish' => $arguments['target'],
             'name' => $arguments['name'],
-            'body' => file_exists($arguments['body']) ? file_get_contents($arguments['body']) : $arguments['body'],
+            'body' => null !== $arguments['body']
+            && file_exists($arguments['body'])
```

Comment:
How about using a variable that holds the result of `null !== $arguments['body'] && file_exists($arguments['body'])` and the using the variable here ? It would be more readable.